_maps/map_files/generic/CentCom.dmm

```diff
@@ -14593,6 +14594,10 @@
 	},
 /turf/open/floor/plasteel/dark,
 /area/tdome/tdomeobserve)
+"GN" = (
+/obj/structure/fans/tiny/invisible,
+/turf/open/indestructible/airblock,
```

Comment:
This is already called an "airblock" turf, what is it not doing that the tiny fan is doing?

Reply:
That turf blocks air through ATMOS_PASS_NO, but active turf processing thinks that this is just a regular open turf next to a space turf. Then it deletes the turf's gases and throws wrong HEY LISTEN errors.

It can be fixed by either placing tiny fans (hacky but works) or rewriting part of AT processing, which I'm not confident enough to do.